api_server.py

```
@@ -502,6 +513,15 @@ def root():
                     available_crew.append(member)
             rules_res = check_rules(available_crew)
 
+            # Check for authentication lock
+            auth_lock = is_auth_locked()
+            last_scrape_time = None
+            if os.path.exists(config.db_path):
+                mtime = os.path.getmtime(config.db_path)
+                # Use project helper for timezone consistency
+                ls_dt = ensure_london(datetime.fromtimestamp(mtime))
```

Ruff (0.15.20)

[warning] `datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)